components/script/script_runtime.rs

```diff
@@ -144,7 +144,7 @@ pub(crate) enum ScriptThreadEventCategory {
 impl From<ScriptThreadEventCategory> for ProfilerCategory {
     fn from(category: ScriptThreadEventCategory) -> Self {
         match category {
-            ScriptThreadEventCategory::AttachLayout => ProfilerCategory::ScriptAttachLayout,
+            ScriptThreadEventCategory::SpawnPipeline => ProfilerCategory::ScriptAttachLayout,
```

Comment:
The profiler category still mentions "attach layout"

Reply:
Thanks for catching that. I thought I had found them all.